src/platform/Darwin/DnssdContexts.cpp

```diff
@@ -516,32 +518,35 @@ void ResolveContext::DispatchSuccess()
     // ChipDnssdResolveNoLongerNeeded don't find us and try to also remove us.
     bool needDelete = MdnsContexts::GetInstance().RemoveWithoutDeleting(this);
 
-    for (auto & interface : interfaces)
-    {
-        auto & ips = interface.second.addresses;
+#if TARGET_OS_TV
+    // On tvOS, priotize results from en0, en1, ir0 in that order, if those
+    // interfaces are present, since those will generally have more up-to-date
+    // information.
+    static const unsigned int priorityInterfaceIndices[] = {
```

Comment:
You could do this explicitly for the non-tvOS case as well, just in case lo0 isn't the lowest interface index for some reason.

Reply:
Done.